### PR TITLE
Fix: [otp modal - api] rename button

### PR DESCRIPTION
**Pull request by akushniruk**

### CODE REVIEW

File: web/src/containers/OpenOrders/index.tsx [hunk 1]
```diff
@@ -135,7 +135,7 @@ export class OpenOrdersContainer extends React.Component<Props> {
                 localeDate(created_at, 'fullDate'),
                 <span style={{ color: setTradeColor(side).color }} key={id}>{Decimal.format(price, priceFixed, ',')}</span>,
                 <span style={{ color: setTradeColor(side).color }} key={id}>{Decimal.format(remainingAmount, amountFixed, ',')}</span>,
-                <span style={{ color: setTradeColor(side).color }} key={id}>{Decimal.format(total, amountFixed, ',')}</span>,
+                <span style={{ color: setTradeColor(side).color }} key={id}>{Decimal.format(+remaining_volume, amountFixed, ',')}</span>,
```
Comment (josadcha) [inline web/src/containers/OpenOrders/index.tsx:138]: but there is still `total` [in headers](https://github.com/openware/baseapp/blob/611ca5a6429ca91eb4b55b3710d59dbd8c3ffc46/web/src/containers/OpenOrders/index.tsx#L87), why did you change value to remaining volume?

**Review by akhlopiachyi — APPROVED** [review]

**Review by josadcha — APPROVED** [review]
